src/custom/pages/Profile/index.tsx

```diff
@@ -55,27 +52,25 @@ export default function Profile() {
   const isTradesTooltipVisible = account && chainId == 1 && !!profileData?.totalTrades
   const hasOrders = useHasOrders(account)
 
-  // const vCowBalance = useTokenBalance(account || undefined, chainId ? V_COW[chainId] : undefined)
-  const vCowBalance = '10,240,800.32'
-  // const cowBalance = useTokenBalance(account || undefined, chainId ? COW[chainId] : undefined)
-  const cowBalance = '0'
+  // Cow balance
+  const cow = useTokenBalance(account || undefined, chainId ? COW[chainId] : undefined)
+
+  // vCow balance values
+  const { unvested, vested, total } = useVCowData()
 
-  // const { vested, total, unvested } = useVCowData()
-  const unvested = '9,240,800.32'
-  const vested = '1,240,800.32'
+  const cowBalance = formatSmartLocaleAware(cow, AMOUNT_PRECISION) || '0'
+  const vCowBalanceVested = formatSmartLocaleAware(vested, AMOUNT_PRECISION) || '0'
+  const vCowBalanceUnvested = formatSmartLocaleAware(unvested, AMOUNT_PRECISION) || '0'
+  const vCowBalance = formatSmartLocaleAware(total, AMOUNT_PRECISION) || '0'
 
-  // TODO: remove once this is not needed anymore
-  // if (isPr || isLocal) {
-  //   console.force.log('vested', formatSmartLocaleAware(vested, vested?.currency.decimals))
-  //   console.force.log('total', formatSmartLocaleAware(total, total?.currency.decimals))
-  //   console.force.log('unvested', formatSmartLocaleAware(unvested, unvested?.currency.decimals))
-  //   console.force.log('cowBalance', formatSmartLocaleAware(cowBalance, cowBalance?.currency.decimals))
-  // }
+  const hasCowBalance = cow && !cow.equalTo(0)
```

Comment:
```suggestion
  const hasCowBalance = !cow?.equalTo(0)
```

Reply:
Actually this doesn't work as expected https://jsfiddle.net/pksz7fy6/

Reply:
Ah ok, I was curious about the negative bool check and undefined